atlasdb-jepsen-tests/src/jepsen/atlasdb.clj

```diff
@@ -1,178 +1,99 @@
+        (info node "Waiting until timelock cluster is ready")
+        (TimestampClient/waitUntilHostReady (name node))
+        (TimestampClient/waitUntilTimestampClusterReady '("n1" "n2" "n3" "n4" "n5"))))
+
+    (teardown! [_ _ node]
+      (c/su
+        (try (c/exec "/atlasdb-timelock-server/service/bin/init.sh" "stop") (catch Exception _))
+        (try (c/exec :rm :-rf "/atlasdb-timelock-server") (catch Exception _))
+        (try (c/exec :rm :-f "/atlasdb-timelock-server.tgz") (catch Exception _))))
 
     db/LogFiles
     (log-files [_ test node]
-      (cond
-        (some #{node} (:nodes test)) ATLASDB_LOG_FILES
-        (some #{node} (:cassandra-nodes)) CASSANDRA_LOG_FILES))))
-
-(defn atlasdb-get [node]
-  (json/parse-string (:body (http/get (format "http://%s:3828/cas" (name node)) {:content-type :json}))))
-
-(defn atlasdb-put! [node new-value]
-  (let [contents (json/generate-string new-value)]
-    (http/put (format "http://%s:3828/cas" (name node)) {:content-type :json :body contents})))
-
-(defn atlasdb-cas! [node old-value new-value]
-  (let [contents (json/generate-string {:oldValue old-value :newValue new-value})]
-    (json/parse-string (:body (http/patch (format "http://%s:3828/cas" (name node)) {:content-type :json :body contents})))))
-
-(defrecord CASClient [node]
-  client/Client
-  (setup! [this test node]
-    (atlasdb-put! node nil)
-    (assoc this :node node))
-
-  (invoke! [this test op]
-    (case (:f op)
-      :read  (try (assoc op :type :ok :value (atlasdb-get node))
-                  (catch Exception e
-                    (warn e "Read failed")
-                    (assoc op :type :fail)))
-
-      :write (do (atlasdb-put! node (:value op))
-                 (assoc op :type :ok))
-      :cas   (let [[value value'] (:value op)
-                   ok? (atlasdb-cas! node value value')]
-               (assoc op :type (if ok? :ok :fail)))))
-
-  (teardown! [_ test]))
-
-(defn cas-client
-  "A compare and set register built around a single consul node."
-  []
-  (CASClient. nil))
-
-(defn r   [_ _] {:type :invoke, :f :read, :value nil})
-(defn w   [_ _] {:type :invoke, :f :write, :value (rand-int 5)})
-(defn cas [_ _] {:type :invoke, :f :cas, :value [(rand-int 5) (rand-int 5)]})
-
-(defmacro with-cassandra
-  "Wraps body in Cassandra setup and teardown."
-  [test & body]
-  `(c/with-ssh (:ssh ~test)
-    (jepsen/with-resources [sessions#
-                            (bound-fn* c/session)
-                            c/disconnect
-                            (:cassandra-nodes ~test)]
-      (let [test# (->> sessions#
-                      (map vector (:cassandra-nodes ~test))
-                      (into {})
-                      (assoc ~test :sessions))]
-        (jepsen/with-os test#
-          (jepsen/with-db test#
-            ~@body))))))
+      ["/atlasdb-timelock-server/var/log/atlasdb-timelock-server-startup.log"])))
```

Comment:
I think there's also a `atlasdb-timelock-server.log` once the server has started